k8s/src/main/scala/io/buoyant/k8s/IngressCache.scala

```diff
@@ -31,34 +33,67 @@ case class IngressSpec(
 case class IngressPath(
   host: Option[String] = None,
   path: Option[String] = None,
+  useCapturingGroups: Option[Boolean],
```

Comment:
this would probably benefit from an explanatory comment.  Specifically, what does it mean for it to be None? (should it be defaulted?)  Is it sufficient to make this a simple boolean that is defaulted to false?